src/layers/activations/softmax.cu

```diff
@@ -101,5 +196,70 @@ void bp_cutoff(int height, int width,
     cutoff);
 }
 
+void max_local_col_entry(int height, int width,
+                         const DataType * __restrict__ input,
+                         int input_ldim,
+                         DataType * __restrict__ workspace,
+                         cudaStream_t stream) {
+  if (width <= 0 || height <= 0) {
```

Comment:
When `width > 0` and `height == 0`., `workspace` should be set with very small values.

Reply:
Done.